### PR TITLE
Pin to 0.10.x toolkit

### PR DESCRIPTION
**Pull request by mattwthompson**
Might be useful - might also be more trouble than its worth as any user of this package probably already knows which version to use.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - openff-toolkit-base
+    - openff-toolkit >=0.10.6,<0.11.0
```
Comment (mattwthompson) [inline recipe/meta.yaml:23]:
```suggestion
    - openff-toolkit-base >=0.10.6,<0.11.0
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: ed682885672af430b7b42b26e9001f5b61db051a5a4cdcf7345ac73ba8980ad3
```
Comment (mattwthompson) [inline recipe/meta.yaml:10]:
```suggestion
  sha256: eec12f25772016f4417f4abf5d2c4db45799d5e1e03eb9dc60e5243ffda4b280
```

```shell
$ curl -sL https://github.com/openforcefield/smirnoff-plugins/archive/0.0.2.tar.gz | openssl sha256
(stdin)= eec12f25772016f4417f4abf5d2c4db45799d5e1e03eb9dc60e5243ffda4b280

**Review by jthorton — APPROVED** [review]

**Comment by mattwthompson** [comment]
Thanks for the review @jthorton - could you please merge as well, as I'm not a maintainer of this feedstock?

**Comment by jthorton** [comment]
@mattwthompson would you like to be added while we are doing this?

**Comment by mattwthompson** [comment]
Sure, happy to help out

**Comment by mattwthompson** [comment]
https://github.com/conda-forge/smirnoff-plugins-feedstock/pull/3

**Pull request merged**